worker/worker-manager/score-worker-manager-impl/src/main/java/org/openscore/worker/management/services/WorkerExecutionMonitorServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package org.openscore.worker.management.services;
+
+import org.apache.log4j.Logger;
+import org.openscore.events.EventBus;
+import org.openscore.events.EventConstants;
+import org.openscore.events.ScoreEvent;
+import org.openscore.worker.management.monitor.ScheduledWorkerMonitor;
+import org.openscore.worker.management.monitor.WorkerMonitors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+
+@Service
+public class WorkerExecutionMonitorServiceImpl implements WorkerExecutionMonitorService{
+    protected static final Logger logger = Logger.getLogger(WorkerExecutionMonitorServiceImpl.class);
+
+    @Autowired
+    private EventBus eventBus;
+
+    @Autowired
+    private WorkerMonitors workerMonitors;
+
+    @Autowired
+    private List<ScheduledWorkerMonitor> scheduledWorkerMonitors;
+
+    @Override
+    public void collectMonitorInformation() {
+        logger.info("Collecting Worker Monitors");
+        try {
+            HashMap<WorkerMonitorInfoEnum, Serializable> monitorInfo = workerMonitors.getMonitorInfo();
+            logger.info("Sending Worker Monitors Info to central:[" + monitorInfo +"]");

```

Comment:
doesn't printing the map just print a reference of the object?
don't you want the values?